unittests/InstructionCountCI/VEX_map3.json

```diff
@@ -5145,14 +5145,23 @@
       ]
     },
     "rorx eax, ebx, 0": {
-      "ExpectedInstructionCount": 2,
-      "Optimal": "No",
+      "ExpectedInstructionCount": 1,
+      "Optimal": "Yes",
       "Comment": [
         "Map 3 0b11 0xf0 32-bit"
       ],
       "ExpectedArm64ASM": [
-        "lsr w20, w7, #0",
-        "ror w4, w20, #0"
+        "lsr w4, w7, #0"
```

Comment:
I don't think this is technically optimal since `lsr` of zero doesn't hit CPU renaming mechanisms.
Maybe we should detect these and convert to a move?

Reply:
Opted for adding the handling into the Bfe IR op